source/application/components/application/index.js

```diff
@@ -118,15 +118,15 @@ Application.propTypes = {
 	themeLoading: t.bool.isRequired
 };
 
-function createLinks(styles, options) {
-	return styles.map(createStyle(options));
+function createLinks(styles) {
+	return styles.map(createStyle());
 }
 
-function createStyle(options) {
+function createStyle() {
 	return style => {
 		return {
 			'rel': 'stylesheet',
-			'href': `${options.base}/style/${style}.css`,
+			'href': `/style/${style}.css`,
```

Comment:
Isn't a proper startBase not superior to absolute styles?
I would tend to reject this and request you to create a correct base depending on the current URL. It may be more work but would probably more stable.

But I don't know all use cases.